package.json

```diff
@@ -95,30 +72,11 @@
     "webpack-merge": "^5.8.0"
   },
   "dependencies": {
-    "@hot-loader/react-dom": "^17.0.1",
-    "@konveyor/lib-ui": "^5.0.0",
-    "@patternfly/react-core": "^4.175.4",
-    "@patternfly/react-icons": "^4.26.4",
-    "@patternfly/react-styles": "^4.25.4",
-    "@patternfly/react-table": "^4.44.4",
-    "@patternfly/react-tokens": "^4.27.4",
-    "axios": "^0.21.2",
-    "dayjs": "^1.10.5",
-    "dayjs-ext": "^2.2.0",
-    "ejs": "^3.1.6",
-    "express": "^4.17.1",
-    "fast-deep-equal": "^3.1.3",
-    "file-saver": "^2.0.5",
-    "get-ssl-certificate": "^2.3.3",
-    "http-proxy-middleware": "^2.0.0",
-    "js-yaml": "^4.1.0",
-    "netmask": "^2.0.2",
-    "node-fetch": "^2.6.1",
-    "q": "^1.5.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-last-location": "^2.0.1",
-    "react-syntax-highlighter": "^15.4.3",
-    "yup": "^0.32.9"
-  }
+    "dayjs": "^1.10.5"
+  },
+  "workspaces": [
+    "pkg/api",
+    "pkg/web",
+    "pkg/qe-tests"
```

Comment:
Even the test suite gets its own workspace.. nice! I'm surprised we didn't need to set nohoist configs for any of these.

Reply:
Interesting, nohoist is not mentioned with npm

Reply:
Ah, might just be a yarn thing. Nvm!